### PR TITLE
changes to support unauthenticated bqm query from portal

### PR DESCRIPTION
**Pull request by kthare10**
https://github.com/fabric-testbed/OrchestratorClient/issues/4

### CODE REVIEW

File: fabric_cf/orchestrator/swagger_client/api/slices_api.py [hunk 1]
```diff
@@ -246,43 +246,45 @@ def slices_delete_slice_id_delete_with_http_info(self, slice_id, **kwargs):  # n
             _request_timeout=params.get('_request_timeout'),
             collection_formats=collection_formats)
 
-    def slices_get(self, **kwargs):  # noqa: E501
+    def slices_get(self, state, **kwargs):  # noqa: E501
```
Comment (ibaldin) [inline fabric_cf/orchestrator/swagger_client/api/slices_api.py:249]: This is more of a comment - what are the kwargs here - presumably you wanted state to be required, that's why it isn't part of kwargs?

**Review by ibaldin — APPROVED** [review]
Left a comment.

File: fabric_cf/orchestrator/swagger_client/api/slices_api.py (hunk 1 above)
Comment (kthare10) [reply fabric_cf/orchestrator/swagger_client/api/slices_api.py:249]: These modules are generated by swagger. I downloaded this code from python client using swagger definition from swagger hub.

Comment (ibaldin) [reply fabric_cf/orchestrator/swagger_client/api/slices_api.py:249]: I c. All good then

**Pull request merged**